### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by voltrevo**
## What is this PR doing?

Adds a template that prefills these headings and the checklist when opening a PR.

## How can these changes be manually tested?

I don't think github gives you a way to do this heh.

## Does this PR resolve or contribute to any issues?

Nope.

## Checklist
- [ ] I have manually tested these changes
- [x] Post a link to the PR in the group chat

## Guidelines
- If your PR is not ready, mark it as a draft
- The `resolve conversation` button is for reviewers, not authors
  - (But add a 'done' comment or similar)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## How can these changes be manually tested?
+
+
+## Does this PR resolve or contribute to any issues?
+
+
+## Checklist
+- [ ] I have manually tested these changes
+- [ ] This PR is ready for review
```
Comment (kautukkundan) [inline .github/pull_request_template.md]: this point feels redundant imo as being a "draft" implies it's not ready for review and once you _un-draft_ a PR it says "marked ready for review" by default.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## What is this PR doing?
+
+
+## How can these changes be manually tested?
```
Comment (kautukkundan) [inline .github/pull_request_template.md:4]: Can remove this point. A CI testing workflow (once it's implemented) should suffice.

**Review by kautukkundan — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (voltrevo) [reply .github/pull_request_template.md]: Ah yeah good point

File: .github/pull_request_template.md (hunk 2 above)
Comment (voltrevo) [reply .github/pull_request_template.md:4]: I'm not so sure about that. Perhaps the wording should change when there is CI but I'd suggest manual testing continues to be a thing. Especially for UI changes in the extension.

Also, do you see any reason for change before CI is actually added?

File: .github/pull_request_template.md (hunk 1 above)
Comment (voltrevo) [reply .github/pull_request_template.md]: Done

(Btw, another suggestion I have is that the author shouldn't click 'resolve conversation' - I think it's up to the person who raised the point to mark it as resolved.)

Comment (kautukkundan) [reply .github/pull_request_template.md]: I Agree. 
Can add this along with _If your PR is not ready, please mark it as a draft_

File: .github/pull_request_template.md (hunk 2 above)
Comment (kautukkundan) [reply .github/pull_request_template.md:4]: yeah sounds good. let's keep it for now.

File: .github/pull_request_template.md (hunk 1 above)
Comment (voltrevo) [reply .github/pull_request_template.md]: Done

**Review by kautukkundan — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**